Raven.Database/Bundles/Replication/Controllers/ReplicationController.cs

```diff
@@ -412,17 +413,21 @@ private void AddSingleConflict(RavenJObject conflict, List<ConflictToResolve> co
             return GetEmptyMessage();
         }
 
-        private void SaveReplicationSource(string src, string lastEtag, int batchSize, string collections = null)
+        private void SaveReplicationSource(string src, string lastEtag, int batchSize, DateTime lastModified, string collections = null)
         {
             var remoteServerInstanceId = Guid.Parse(GetQueryStringValue("dbid"));
 
             var replicationDocKey = Constants.RavenReplicationSourcesBasePath + "/" + remoteServerInstanceId;
 
             var replicationDocument = Database.Documents.Get(replicationDocKey, null);
             var lastAttachmentId = Etag.Empty;
+            DateTime lastModification = lastModified;
             if (replicationDocument != null)
```

Comment:
What about deletion? If we have a cluster that is idle for a while, but then we delete a bunch of stuff?